src/Events/Program.cs

```diff
@@ -7,14 +7,17 @@
 using Altinn.Common.AccessToken;
```

Comment:
Ref github actions analysis, CogComplexity, should we consider moving some logic to a Startup.cs class?

Reply:
We consolidated these two files when migrating to .net 6. Thinking the complexity will be the same if we move it to a Startup.cs. Creating extension methods for registration of services might help with the error messages, but not sure if it is less or more complex to follow the code then. Would suggest we create an issue to analyse, but leave this file as is for now